core/src/test/resources/Pipes/1.txt

```diff
@@ -0,0 +1,3 @@
+function giveNumber(x) {
+    return x * x;
+}
```

Comment:
This is javascript, please name the file accordingly, eg. /javascript/rhino-tests.js

Reply:
done